app/javascript/components/admin/AdminTable.js

```diff
@@ -465,6 +495,16 @@ class AdminTable extends React.Component {
     );
   };
 
+  removeFormulaStart = value => {
```

Comment:
this seems like it would be a good utils function. can you create a new file in the `/utils` folder called `Export.js` and add this there? Also add a method comment so people know what it's for